tests/mock_tables/dbconnector.py

```diff
@@ -49,23 +47,34 @@ def load_database_config():
         sonic_db_file_path=os.path.join(
             os.path.dirname(os.path.abspath(__file__)), 'database_config.json'))
 
+class MockSonicV2Connector(SonicV2Connector_Native):
```

Comment:
Because swsscommon ow using c++ version SonicDBConfig, so python method clean_up_config() does not work anymore.

Reply:
Change in mock reverted, so UT will still be using the swsssdk for mock.